### PR TITLE
Documentation for existing Kubernetes projects.

### PR DESCRIPTION
**Pull request by ivanporty**
Working on the animated GIF for the live template, meanwhile ready for doc itself review.

### CODE REVIEW

**Comment by etanshaul** [comment]
nice! 

I know I suggested this before and I was convinced to hold off. but this page is getting rather long. wdyt about (in a separate PR) splitting it up. e.g. in the tutorial section have a link to the hello-world guide which lives on its own page. same for this section.

**Comment by ivanporty** [comment]
It made sense to keep it in one piece on the main page of the previous repo since people tend to stay on it most of time. Since we are already nested, I agree, we can split this. I'll do that in the next PR.

File: kubernetes/README.md [hunk 1]
```diff
@@ -154,5 +154,32 @@ You can use the other Kubernetes run target to build the image and deploy the pr
 
 ## Setup existing Kubernetes projects with the plugin
 
-(coming soon)
-
+You can open any project with a configured container image build (using Dockerfile or [Jib](https://github.com/GoogleContainerTools/jib)) and Kubernetes manifests, and use it with the plugin. The only additional bit of configuration is a Skaffold YAML file that you can create from a provided template. Here is a quick tutorial:
```
Comment (etanshaul) [inline kubernetes/README.md]: suggestion:

`You can open any project with a configured container image build (using Dockerfile or Jib) and Kubernetes manifests, and use it with the plugin.`

->

`You can open any existing project already configured with Kubernetes manifests and Dockerfile (or Jib), and use it with the plugin.`

**Comment by etanshaul** [comment]
looks great. no more comments other than the one.

**Pull request merged**